README.md

```diff
@@ -1,2 +1,72 @@
 # patch-filters
 special project to host GSOC project Patch Suppressions 
+
+## PatchFilter Report Setup
+
+### Requirements
+
+- Checkstyle repository which including patch-filter branch need to be cloned.
+Because patch-filter has not been merged by Checkstyle, so you should fork Checkstyle from [here](https://github.com/HuGanghui/checkstyle)
```

Comment:
I don't understand why you use SuppressionPatchFilter from your own repository, because you should test SuppressionPatchFilter from this repository instead.
You should use main checkstyle and add patch-filters jar to classpath, so SuppressionPatchFilter will be visible when checkstyle runs.

Reply:
ok, I will try the jar format, then give response